code/formal_informal.py

```diff
@@ -0,0 +1,42 @@
+import pandas as pd 
+import numpy as np 
+from matplotlib import pyplot as plt 
+import os 
+from transformers import pipeline
+
+data1 = pd.read_csv(r'C:\Users\rasik\Documents\Independent Study\data\translated data\1_15_final.csv')
```

Comment:
- use better names for the variable. 
- I think the dataset may be better organized in tidy data format in a single dataset with the size of the server as a variable. Then you can filter them to get different groups. 
- don't use the absolute path that differ across team members. use symlink so that everyone can replicate without changing the code. see https://yyiki.org/wiki/Research%20workflow/Data%20management/ (in progress)

Reply:
Variables names have been updated for this python file. I created a single dataset and will be using that for all the analysis based python notebooks. Shifting everything to relative path rather than absolute